memory/src/main/scala/filodb.memory/format/vectors/DoubleVector.scala

```diff
@@ -271,11 +297,15 @@ extends AppendableVectorWrapper[Long, Double] {
  * A wrapper to return Doubles from a Long vector... for when one can compress Double vectors as DDVs
  */
 object DoubleLongWrapDataReader extends DoubleVectorDataReader {
+  class DoubleLongWrapIterator(innerIt: LongIterator) extends DoubleIterator {
```

Comment:
How are NaNs handled in the Double data stored as Long vector ?

Reply:
NaNs are not integral so cannot take advantage of Long based DDV compression.   Today any vectors with NaNs will not be compressed at all.